plugins/pencilblue/controllers/index.js

```diff
@@ -275,10 +275,15 @@ Index.prototype.renderContent = function(content, contentSettings, themeSettings
     ats.registerLocal('author_position', content.author_position ? content.author_position : '');
     ats.registerLocal('media_body_style', content.media_body_style ? content.media_body_style : '');
     ats.registerLocal('comments', function(flag, cb) {
-       if (isPage || !contentSettings.allow_comments) {
+        if (isPage || !contentSettings.allow_comments) {
            cb(null, '');
            return;
-       }
+        }
+
+		if (!content.allow_comments) {
+			cb(null, '');
+			return;
+		}
```

Comment:
This is really nit-picky but as I've come across them I've been trying to do ```return cb(null, '');``` to try and save a line.  Would you mind changing that?